packages/angular_devkit/build_angular/src/webpack/plugins/optimize-css-webpack-plugin.ts

```diff
@@ -133,7 +133,7 @@ export class OptimizeCssWebpackPlugin {
               map,
             );
           } else {
-            newSource = new RawSource(output.css);
+            newSource = new sources.RawSource(output.css);
           }
 
           compilation.assets[file] = newSource;
```

Comment:
Just note, we have https://github.com/webpack-contrib/css-minimizer-webpack-plugin - cache and multi threading included

Reply:
Thanks for sharing, will take a look